qutebrowser/misc/crashdialog.py

```diff
@@ -624,3 +642,8 @@ def dump_exception_info(exc, pages, cmdhist, objects):
     print('\n'.join(cmdhist), file=sys.stderr)
     print("\n---- Objects ----", file=sys.stderr)
     print(objects, file=sys.stderr)
+    print("\n---- Environment ----", file=sys.stderr)
+    try:
+        print(_get_environment_vars(), file=sys.stderr)
+    except Exception:
+        traceback.print_exc()
```

Comment:
No need for the `try`/`except` here as it's console output anyways, and only intended for debugging (this code path is only run on the CI server)